superdesk/io/feeding_services/ftp.py

```diff
@@ -79,8 +97,14 @@ def _update(self, provider, update):
             except KeyError:
                 logger.warning('missing move_path')
                 move_dest_path = None
+            try:
+                move_dest_path_error = os.path.join(config.get('path', ''), config['move_path_error'])
```

Comment:
it has not been requested, I just thought one may want to move on success but not on error, but I may be wrong.